src/main.cpp

```diff
@@ -78,240 +85,238 @@ SharedProperty<data_t> turnRightVal(0);
-		// Throttle Data
-		const throttle_t throttleData[] = { throttleVal.value() };
-		can.write(CANMessage(CAN_STEERING_THROTTLE, throttleData, 1));
-
-		// Ready Data
-		char ready = (brakeVal.value() << 2) | (dmsVal.value() << 1) | ignitionVal.value();
-		const char readyData[] = { ready };
-		can.write(CANMessage(CAN_STEERING_READY, readyData, 1));
+        // Ready Data
+        char ready = (brakeVal.value() << 2) | (dmsVal.value() << 1) | ignitionVal.value();
+        const char readyData[] = {ready};
+        can.write(CANMessage(CAN_STEERING_READY, readyData, 1));
 
         timerMotor.reset();
-	}
+    }
 }
 
 throttle_t get_throttle_val() {
+    int throttleVal = (int)(throttle.read() * 10000);
 
-	int throttleVal = (int)(throttle.read() * 10000);
-
-	#ifdef DEBUG_MODE
-		printf("Throttle Input: %04d - ", throttleVal);
-	#endif
+#ifdef DEBUG_MODE
+    printf("Throttle Input: %04d - ", throttleVal);
+#endif
 
-	// Keep values within min/max
-	if (throttleVal <= MIN_THROTTLE_INPUT) {
-		throttleVal = MIN_THROTTLE_INPUT;
-	} else if (throttleVal >= MAX_THROTTLE_INPUT) {
-		throttleVal = MAX_THROTTLE_INPUT;
-	}
+    // Keep values within min/max
+    if (throttleVal <= MIN_THROTTLE_INPUT) {
+        throttleVal = MIN_THROTTLE_INPUT;
+    } else if (throttleVal >= MAX_THROTTLE_INPUT) {
+        throttleVal = MAX_THROTTLE_INPUT;
+    }
 
-	// Normalize throttle value
-	throttleVal -= MIN_THROTTLE_INPUT;
+    // Normalize throttle value
+    throttleVal -= MIN_THROTTLE_INPUT;
 
-	// Scale for output
-	throttle_t throttleOut = (throttleVal * MAX_THROTTLE_OUTPUT) / THROTTLE_RANGE;
+    // Scale for output
+    throttle_t throttleOut = (throttleVal * MAX_THROTTLE_OUTPUT) / THROTTLE_RANGE;
 
-	return throttleOut;
+    return throttleOut;
 }
 
 data_t getDmsVal() {
-	int dmsCtrl = (int)(dms.read() * 10000);
-	dmsLed.write(1);
-	wait_us(40);
+    int dmsCtrl = (int)(dms.read() * 10000);
+    dmsLed.write(1);
+    wait_us(40);
     int dmsVal = (int)(dms.read() * 10000);
-	dmsLed.write(0);
-	int dmsDelta = dmsVal - dmsCtrl;
-	#ifdef DEBUG_MODE
-		printf("DMS Delta: %04d\n", dmsDelta >= 0 ? dmsDelta : 0);
-	#endif
+    dmsLed.write(0);
+    int dmsDelta = dmsVal - dmsCtrl;
+#ifdef DEBUG_MODE
+    printf("DMS Delta: %04d\n", dmsDelta >= 0 ? dmsDelta : 0);
+#endif
 
-    return  (data_t)(dmsDelta > DMS_DELTA_THRESHOLD);
+    return (data_t)(dmsDelta > DMS_DELTA_THRESHOLD);
 }
 
-// Checks for gps speed / bms data updates from telemetry
+// Checks for CAN updates
 void receive_can() {
-	CANMessage msg;
-
-	if (can.read(msg)) {
-		if (msg.id == CAN_TELEMETRY_GPS_DATA) {
-			currentSpeedVal.set((speed_t)msg.data[0]);
-		} else if (msg.id == CAN_TELEMETRY_BMS_DATA) {
-			batt_t soc, voltage;
-			memcpy((void*)&soc, (void*)msg.data, 4);
-			memcpy((void*)&voltage, (void*)(msg.data + 4), 4);
-			batterySocVal.set(soc);
-			batteryVoltageVal.set(voltage);
-		}
-	}
+    CANMessage msg;
+
+    if (can.read(msg)) {
+        if (msg.id == CAN_TELEMETRY_GPS_DATA) {
```

Comment:
I noticed that CAN_TELEMETRY_GPS_DATA isn't part of can-common.  I will make a note to add it

Reply:
It is, have you pulled the latest from that repo?

Reply:
I checked the repo but must have missed it